### PR TITLE
Adds frequency sort feature to PosTagVisualizer (#771)

### PR DESCRIPTION
**Pull request by ajaysub110**
I've added a option to sort the PosTagVisualizer plot by frequency of tags in the text corpus.

### CODE REVIEW

**Comment by lwgray** [comment]
@ajaysub110 thank you for putting this together. We are really excited about implementing this. However, we are a bit busy over the next 2 weeks and can fully review it afterwards.

**Comment by ajaysub110** [comment]
@lwgray Sure. Thank you for your response. 
Apart from this, I'm interested in contributing to yellowbrick as part of GSoC 2019. I would like to be added to any mailing list or IRC that exists for the purpose of project discussions. Also, how do I get in touch with the GSoC mentors?

**Comment by wagner2010** [comment]
@ajaysub110 Thanks for your PR and your interest in Yellowbrick generally.  As you may know, the student application period for GSoC is March 25-April 9th.  So In terms of positioning yourself for the process you should focus on your [student application](https://summerofcode.withgoogle.com/how-it-works/).  But we welcome your communication here at our GitHuB page in the form of pull requests (we have sufficient number of issues that we are solving but if you have any solutions or partial solutions please submit a PR).  It would also be advisable to check out our [docs,](http://www.scikit-yb.org/en/latest/) follow us @scikit-yb on Twitter and star us here on GitHub.  Our docs has a contributing section, a quick start guide, and walk through of all the visualizers that we currently have so I would recommend walking through all of those docs.  Cheers!

**Comment by ajaysub110** [comment]
@wagner2010 Thanks for your advice! I'll definitely try and contribute to yellowbrick in the mean time, and focus on my application during the application period.

**Comment by ajaysub110** [comment]
> In addition to those notes, I'm wondering what you think about adding a check in the `finalize` method that checks to see if the frequency attribute is true, and if so, modifies the x-axis label to something like `"{} part-of-speech tags, sorted by frequency"`.

Sounds good. I'll add that to the PR right away!

File: yellowbrick/text/postag.py [hunk 1]
```diff
@@ -375,15 +409,10 @@ def postag(X, ax=None, tagset="penn_treebank", colormap=None,
         Returns the axes on which the PosTagVisualizer was drawn.
     """
     # Instantiate the visualizer
-    visualizer = PosTagVisualizer(
-        ax, tagset, colors, colormap, **kwargs
-    )
+    visualizer = PosTagVisualizer(ax, tagset, colors, colormap, **kwargs)
```
Comment (rebeccabilbro) [inline yellowbrick/text/postag.py]: still need to add `frequency` here?

**Comment by rebeccabilbro** [comment]
Hi there @ajaysub110 &mdash; can you please go ahead and add the new keyword argument to the quick method so that we can get your PR merged in? We're nearly there!

**Comment by rebeccabilbro** [comment]
@ajaysub110 — just wanted to check that you're still planning to finish this PR? We're very close but we can't merge this in until it is complete, so  the quick method must be updated first. Please let me know if you have any questions!

**Comment by bbengfort** [comment]
For the failed test, the original `test_frequency_mode` produced a baseline image that looked like:

test_frequency_mode

After my update to ensure the quick method used the frequency param it looks like this:

test_frequency_mode

I believe this is a correct result (e.g. they are now sorted) and I've fixed the tests, but wanted to make sure by placing these images here for conversation.

**Comment by rebeccabilbro** [comment]
Thank you @bbengfort for getting this the rest of the way across the finish line! Yes, that last test image looks like what I think we're expecting to see. I'll go ahead and merge this in as soon as the CI tests finish running!

**Pull request merged**